reminders-menubar/Views/Helpers/RmbHighlightedTextField.swift

```diff
@@ -41,6 +41,22 @@ struct RmbHighlightedTextField: NSViewRepresentable {
         return attributedString
     }
     
+    private func getPlaceholderAttributedString(from text: String) -> NSMutableAttributedString {
+        let fullRange = text.fullRange
+        
+        let attributedString = NSMutableAttributedString(string: text)
+        attributedString.beginEditing()
+        attributedString.addAttribute(.foregroundColor,
+                                       value: NSColor.systemGray.withAlphaComponent(0.5),
+                                       range: fullRange)
+        attributedString.addAttribute(.font,
+                                      value: NSFont.preferredFont(forTextStyle: .callout),
+                                      range: fullRange)
+        attributedString.endEditing()
+        
+        return attributedString
+    }
```

Comment:
As the attributes for the placeholder will always be applied to the fullRange we can use the NSAttributedString constructor to be cleaner.

```suggestion
    private func getPlaceholderAttributedString(from text: String) -> NSAttributedString {
        let attributes: [NSAttributedString.Key: Any] = [
            .foregroundColor: NSColor.systemGray.withAlphaComponent(0.5),
            .font: NSFont.preferredFont(forTextStyle: .callout)
        ]
        
        return NSAttributedString(string: text, attributes: attributes)
    }
```